finalPrep/modZeroHero.js

```diff
@@ -56,29 +81,65 @@ var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.
 //   - luckyNumbers (array)
 //   - address (object with following key/values: number , street , state, zip)
 
-
+var scaryMonster = {
+  name: 'Beth',
+  smell: 'sickly sweet',
+  weight: 3002,
+  citiesDestroyed: ['Montpillier, VT', 'Rifle, CO', 'Limon, CO'],
+  luckyNumbers: [1,4,95],
+  address: {
+    number: 626,
+    street: 'Monster way',
+    state: 'CO',
+    zip: 80111
+  }
+};
+console.log(scaryMonster);
 // Create a new class called SuperHero
 // - Your class should have the following DYNAMIC values
-//   - name 
+//   - name
 //   - superpower
-//   - age 
+//   - age
 // - Your class should have the following STATIC values
 //   - archNemesis, assigned to "The Syntax Error"
 //   - powerLevel = 100
-//   - energyLevel = 50 
+//   - energyLevel = 50
+class Superhero {
+  constructor(name, superpower, age, archNemesis, powerLevel, energyLevel) {
```

Comment:
According to the instructions, `archNemesis`, `powerLevel`, and `energyLevel` are assigned to static values - thus you don't need to pass those as parameters in the constructor function.